pkg/antctl/raw/traceflow/command.go

```diff
@@ -0,0 +1,395 @@
+			(*pkt).TransportHeader.TCP = new(v1alpha1.TCPHeader)
+		}
+		(*pkt).TransportHeader.TCP.SrcPort = int32(r)
+	}
+	r, err = getFieldPortValue(cleanFlow, "tcp_dst")
+	if err != nil {
+		return nil, fmt.Errorf("error when get tcp_dst value: %w", err)
+	}
+	if r != 0 {
+		if (*pkt).TransportHeader.TCP == nil {
+			(*pkt).TransportHeader.TCP = new(v1alpha1.TCPHeader)
+		}
+		(*pkt).TransportHeader.TCP.DstPort = int32(r)
+	}
+	r, err = getFieldPortValue(cleanFlow, "udp_src")
+	if err != nil {
+		return nil, fmt.Errorf("error when get udp_src value: %w", err)
+	}
+	if r != 0 {
+		if (*pkt).TransportHeader.UDP == nil {
+			(*pkt).TransportHeader.UDP = new(v1alpha1.UDPHeader)
+		}
+		(*pkt).TransportHeader.UDP.SrcPort = int32(r)
+	}
+	r, err = getFieldPortValue(cleanFlow, "udp_dst")
+	if err != nil {
+		return nil, fmt.Errorf("error when get udp_dst value: %w", err)
+	}
+	if r != 0 {
+		if (*pkt).TransportHeader.UDP == nil {
+			(*pkt).TransportHeader.UDP = new(v1alpha1.UDPHeader)
+		}
+		(*pkt).TransportHeader.UDP.DstPort = int32(r)
+	}
+
+	return pkt, nil
+}
+
+func getFieldPortValue(cleanFlow string, f string) (int, error) {
+	for _, v := range strings.Split(cleanFlow, ",") {
+		m, err := regexp.MatchString(fmt.Sprintf("%s=[0-9]+", f), v)
+		if err != nil {
+			return 0, nil
+		}
+		if m {
+			r, err := strconv.Atoi(v[len(f)+1:])
+			if err != nil {
+				return 0, err
+			}
+			return r, nil
+		}
+	}
+	return 0, nil
+}
+
+func output(tf *v1alpha1.Traceflow) error {
+	r := Response{
+		Name:        tf.Name,
+		Phase:       tf.Status.Phase,
+		Source:      fmt.Sprintf("%s/%s", tf.Spec.Source.Namespace, tf.Spec.Source.Pod),
+		Destination: tf.Spec.Destination.IP,
+		NodeResults: tf.Status.Results,
+	}
+	if len(tf.Spec.Destination.IP) == 0 {
+		if len(tf.Spec.Destination.Service) != 0 {
+			r.Destination = fmt.Sprintf("%s/%s", tf.Spec.Destination.Namespace, tf.Spec.Destination.Service)
+		} else {
+			r.Destination = fmt.Sprintf("%s/%s", tf.Spec.Destination.Namespace, tf.Spec.Destination.Pod)
+		}
+	}
+	if option.outputType == "json" {
+		if err := jsonOutput(&r); err != nil {
+			return fmt.Errorf("error when converting output to json: %w", err)
+		}
+	} else if option.outputType == "yaml" {
+		if err := yamlOutput(&r); err != nil {
+			return fmt.Errorf("error when converting output to yaml: %w", err)
+		}
+	} else {
+		return fmt.Errorf("output types are yaml and json")
+	}
+	return nil
+}
+
+func yamlOutput(r *Response) error {
+	o, err := yaml.Marshal(&r)
+	if err != nil {
+		return err
+	}
+	fmt.Println(string(o))
+	return nil
+}
+
+func jsonOutput(r *Response) error {
+	o, err := json.Marshal(r)
+	if err != nil {
+		return err
+	}
+	var b bytes.Buffer
+	if err = json.Indent(&b, o, "", "  "); err != nil {
+		return err
+	}
+	fmt.Println(string(b.Bytes()))
+	return nil
+}
+
+func getTFName(prefix string) string {
+	if !option.waiting {
+		return prefix
+	}
+	var lettersAndDigits = []rune("abcdefghijklmnopqrstuvwxyz0123456789")
```

Comment:
Looks like we didn't set up a random seed, will the randIdx always the same on the same machine.  Maybe we should set up a random seed in cmd/antctl/antctl.go.

Reply:
You are right. Add random seed in antctl main.